### PR TITLE
extend OCL to support service actions.

### PR DESCRIPTION
**Pull request by WangLiNaruto**
eclipse-xpanse/xpanse#2207

### CODE REVIEW

**Comment by yuanxiaojun987** [comment]
LGTM

**Comment by Alice1319** [comment]
LGTM

File: middleware/HuaweiCloud-Kafka-autofill.yml [hunk 1]
```diff
@@ -251,6 +251,57 @@ serviceConfigurationManage:
...
+          playbookName: "middleware/ansible/kafka/kafka-container-manage.yml"
+          virtualEnv: ""
+          pythonVersion: "3.10"
+          repoUrl: https://github.com/eclipse-xpanse/xpanse-samples
+          branch: main
+          requirementsFile: "requirements.txt"
+          galaxyFile: ""
+          ansibleInventoryRequired: false
+    actionParameters:
+      - name: kafka_cfg_message_max_bytes
```
Comment (swaroopar) [inline middleware/HuaweiCloud-Kafka-autofill.yml]: we need some other action parameters which are really valid for upgrade of a kafka broker.

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

**Comment by baixinsui** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**